CHANGELOG.md

```diff
@@ -15,6 +15,99 @@ and includes an additional section for migration notes.
 
 
 ## [Unreleased]
+
+
+### Added
+- *ORCA-161* Added dead letter queue and cloudwatch alarm terraform code to recovery SQS queue.
+
+### Changed
+- *ORCA-161* Changed staged recovery SQS queue type from FIFO to standard queue.
+
+### Migration Notes
+
+- The user should update their `orca.tf`, `variables.tf` and `terraform.tfvars` files with new variable. The following required variable has been added:
+  - db_admin_username (defaults to "postgres")
+  
+- Add the following ORCA required variables definition to your `variables.tf` or `orca_variables.tf` file.
+
+```terraform
```

Comment:
Same duplicate-doc issue with a lot of this.

Reply:
removed duplicates